### PR TITLE
add cinttypes to cr_signal.cc

### PR DESCRIPTION
**Pull request by zhouxindong**

### CODE REVIEW

**Comment by xkszltl** [comment]
Add some description for why your're doing this. You can edit the first msg.

File: src/cr_signal.cc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include <glog/logging.h>
+#include <cinttypes>
```
Comment (xkszltl) [inline src/cr_signal.cc:2]: Put it together with other STL headers and sort them alphabetically.

**Review by xkszltl — CHANGES_REQUESTED** [review]

File: src/cr_signal.cc (hunk 1 above)
Comment (zhouxindong) [reply src/cr_signal.cc:2]: fine

**Comment by zhouxindong** [comment]
I will fix according reviews, and PR again later

**Comment by xkszltl** [comment]
@zhouxindong You don't need to open a new PR. Simply update the branch and it will reflect in this PR.

**Pull request closed without merging**